test/test_helper.rb

```diff
@@ -1,4 +1,5 @@
 require "bundler/setup"
+require "pp"
```

Comment:
Is this actually needed?

Reply:
I find it really helpful and almost always add it to test helper. It isn't needed for this PR, but just in general. I always find it a little annoying to have to require it on a case by case basis each time I want to debug something.